tests/ltp/ramfs_tests_other_errors.txt

```diff
@@ -558,7 +558,6 @@
 /ltp/testcases/kernel/syscalls/sched_get_priority_min/sched_get_priority_min02
 /ltp/testcases/kernel/syscalls/sched_getaffinity/sched_getaffinity01
 /ltp/testcases/kernel/syscalls/sched_getattr/sched_getattr02
-/ltp/testcases/kernel/syscalls/sched_getparam/sched_getparam01
 /ltp/testcases/kernel/syscalls/sched_getparam/sched_getparam03
```

Comment:
what is to stop test 03 from being enabled too?

Reply:
test /ltp/testcases/kernel/syscalls/sched_getparam/sched_getparam03 passes with the file /proc/sys/kernel/pid_max (can verify by applying changes in PR - #1013). But this PR still needs to address tid cycling as noted by comment in threads.c. Once we figure that out, we can enable it too.

Reply:
thanks.